xarray/core/variable.py

```diff
@@ -1049,6 +1064,12 @@ def chunk(self, chunks={}, name=None, lock=False):
             )
             chunks = {}
 
+        if isinstance(chunks, (numbers.Number, str, int, tuple, list)):
```

Comment:
My understanding is that `Number` is not good for typing: https://github.com/python/mypy/issues/3186, https://stackoverflow.com/questions/50928592/mypy-type-hint-unionfloat-int-is-there-a-number-type

```suggestion
        if isinstance(chunks, (float, str, int, tuple, list)):
```